tests/translator/input/basic_function.yaml

```diff
@@ -50,37 +50,37 @@ Resources:
       Timeout: 60
       VpcConfig:
         SecurityGroupIds:
-        - sg-edcd9784
+          - sg-edcd9784
```

Comment:
If we get the previous version of this, will the validation fail?

Reply:
It passes, just tested.
This was changed by the VSCode save auto-format.